src/server_process.js

```diff
@@ -57,38 +57,12 @@ function initialize_wallet(config, done) {
 
   var bitcoinAsset = wallet.getAssetDefinitionByMoniker('bitcoin');
   console.log('My Bitcoin address:', wallet.getSomeAddress(bitcoinAsset));
-  var euroAsset = wallet.getAssetDefinitionByMoniker('euro2');
+  var euroAsset = wallet.getAssetDefinitionByMoniker('euro');
   console.log('My Euro address:', wallet.getSomeAddress(euroAsset, true));
 
   wallet.once('syncStop', function () {
-    return Q.ninvoke(wallet, 'getBalance', bitcoinAsset)
```

Comment:
This stuff is actually needed: we need many coins to be able to process requests in parallel. 

Is it causing problems?